utils/mail.go

```diff
@@ -162,7 +162,7 @@ func (e *Email) Bytes() ([]byte, error) {
 
 // AttachFile Add attach file to the send mail
 func (e *Email) AttachFile(args ...string) (a *Attachment, err error) {
-	if len(args) < 1 && len(args) > 2 {
+	if len(args) < 1 || len(args) > 2 {
```

Comment:
Since `args` here is expecting exactly two parameters, you could drop the check and explicitly specify the accepted arguments. Example:
```golang
 func (e *Email) AttachFile(name string, id string) (a *Attachment, err error) {
        f, err := os.Open(filename)
	if err != nil {
		return
	}
	ct := mime.TypeByExtension(filepath.Ext(filename))
	basename := path.Base(filename)
	return e.Attach(f, basename, ct, id)
}
```